runtime/acala/src/lib.rs

```diff
@@ -1745,8 +1745,8 @@ pub type SignedExtra = (
 	frame_system::CheckEra<Runtime>,
 	runtime_common::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
-	module_transaction_payment::ChargeTransactionPayment<Runtime>,
 	module_evm::SetEvmOrigin<Runtime>,
+	module_transaction_payment::ChargeTransactionPayment<Runtime>,
```

Comment:
add comment explains that `SetEvmOrigin` needs to be above `ChargeTransactionPayment` so it can process ERC20 transfer